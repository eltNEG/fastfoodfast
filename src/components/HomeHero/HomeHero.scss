@import '../../styles/styles';

.home-hero {
  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('../../assets/images/fff-hero.png') no-repeat center;
  background-size: cover;
  width: 100%;
  height: 20rem;
  color: $white;
  padding: 5rem;
  position: relative;
  .home-hero-content {
    position: absolute;
    opacity: 1;
    filter: brightness(300%);
    color: $white;
    font-weight: 600;
    min-width: 18rem;
    width: 50%;
    max-width: 23rem;
    text-align: justify;
    line-height: 2.4rem;
    color: rgb(255, 255, 255);
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    align-content: center;
    div {
      font-size: 2rem;
      width: 100%;
      margin-bottom: 1rem;
      text-align: justify;
    }
    input {
      outline: none;
      border: 0;
      border: solid 1px $white;
      border-radius: $border-radius;
      height: 2.5rem;
      width: 60%;
      background-color: rgba(255, 255, 255, 0);
      padding-left: 1rem;
    }
    button {
      width: 30%;
      background-color: $white;
      color: $dark-blue;
      font-weight: 600;
      border-radius: $border-radius;
    }
  }
}