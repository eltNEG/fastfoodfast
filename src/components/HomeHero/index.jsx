import React from 'react';
import './HomeHero.scss';

export default () => (
  <div className="home-hero">
    <div className="home-hero-content">
      <div>Your Next Delicious Meal is Just a Few Clicks Away</div>
      <input placeholder='"Try Shawarma"' />
      <button>Search</button>
    </div>
  </div>
);
