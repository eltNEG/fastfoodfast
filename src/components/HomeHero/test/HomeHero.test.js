import React from 'react';
import { shallow } from 'enzyme';
import HomeHero from '..';

describe('HomeHero snapshot', () => {
  it('should match HomeHero.jsx snapshot', () => {
    const wrapper = shallow(<HomeHero />);
    expect(wrapper).toMatchSnapshot();
  });
});