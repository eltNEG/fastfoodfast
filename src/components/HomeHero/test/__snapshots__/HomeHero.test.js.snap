// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHero snapshot should match HomeHero.jsx snapshot 1`] = `
<div
  className="home-hero"
>
  <div
    className="home-hero-content"
  >
    <div>
      Your Next Delicious Meal is Just a Few Clicks Away
    </div>
    <input
      placeholder="\\"Try Shawarma\\""
    />
    <button>
      Search
    </button>
  </div>
</div>
`;
